src/exercise5.js

```diff
@@ -0,0 +1,17 @@
+function dec2bin(num) {
+  return (num >>> 0).toString(2);
+}
```

Comment:
I think the function name is overshortened and can be confusing  `dec` and `bin` can be interpreted differently and I would not rename `to` to be `2` just to save one character.

A more descriptive name would be better